main/api/v1/auth.py

```diff
@@ -28,9 +28,10 @@ def post(self):
     if not username or not password:
       return flask.abort(400)
 
-    if username.find('@') > 0:
+    user_db = None
+    if '@' in username:
       user_db = model.User.get_by('email', username.lower())
-    else:
+    if not user_db:
       user_db = model.User.get_by('username', username.lower())
```

Comment:
I think that it will be more efficient (because it does not allocate memory for temporary array):
``` python
user_db = model.User.get_by('email' if '@' in username else 'username', username.lower())
```

test:
``` python
#!/usr/bin/env python
from timeit import timeit

TEST_DATA='AaAaAaAa@example.com'


def test_1():
  x = ['username', 'email']['@' in TEST_DATA]

def test_2():
  x = 'email' if '@' in TEST_DATA else 'username'


print timeit(test_1, number=1000000)
print timeit(test_2, number=1000000)
```
result:
``` shell
0.251525163651
0.14129281044
```